src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1887,6 +1901,20 @@ void layout_optimizer::select_preferred_formats_for_onednn(program_node& node, d
                 }
             }
 
+            // Optimize permute if input is convolution and next is output.
```

Comment:
* `next is output`? or `next permute can be fused into conv`?
* could you write more detailed description?

Reply:
Updated